### PR TITLE
Adding MySQL to the documentation

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
Adding MySQL installation and configuration to  Ubuntu.md and Personal Server configuration

### CODE REVIEW

File: website/site/content/guide/admin/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Personal Server settings are stored in `config.json` and is read when the server
 | -----------   | ----------- | ---
 | serverRoot    | Root URL of the server    | "http://localhost:8000"
 | port          | Server port               | 8000
-| dbtype        | Type of database. `sqlite3` or `postgres` | "sqlite3"
+| dbtype        | Type of database. `sqlite3` or `postgres` or `mysql` | "sqlite3"
```
Comment (jespino) [inline website/site/content/guide/admin/_index.md:16]:
```suggestion
| dbtype        | Type of database. `sqlite3`, `postgres` or `mysql` | "sqlite3"
```

File: website/site/content/download/personal-edition/ubuntu.md [hunk 2]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
 "dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
 ```
 
+## Install MySQL 
+
+Alternate to Postgress you also can store your data in a MySQL/MariaDB database.
+16). To install, run:
```
Comment (jespino) [inline website/site/content/download/personal-edition/ubuntu.md:153]: What is this `16).` here?

**Review by jespino — CHANGES_REQUESTED** [review]
To tiny changes needed, otherwise looks good to me. Thanks @ctlaltdieliet! :tada:

File: website/site/content/download/personal-edition/ubuntu.md (hunk 2 above)
Comment (ctlaltdieliet) [reply website/site/content/download/personal-edition/ubuntu.md]: Some copy-paste error.

**Review by jespino — APPROVED** [review]
LGTM

File: website/site/content/download/personal-edition/ubuntu.md [hunk 3]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
...
+Edit the Focalboard config.json:
+
+```
+nano /opt/focalboard/config.json
+```
+
+Change the dbconfig setting to use the MySQL database you created:
+```
+"dbtype": "mysql",
+"dbconfig": "mysql://boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
```
Comment (3l0w) [inline website/site/content/download/personal-edition/ubuntu.md:185]: Accoding to my test & #301 this shoud be:
```
"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
```

**Review by 3l0w — CHANGES_REQUESTED** [review]

File: website/site/content/download/personal-edition/ubuntu.md (hunk 3 above)
Comment (ctlaltdieliet) [reply website/site/content/download/personal-edition/ubuntu.md]: Indeed! Thanks for noticing

**Review by ctlaltdieliet — COMMENTED** [review]
removing mysql:// from dbconfig parameter

File: website/site/content/download/personal-edition/ubuntu.md [hunk 4]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
 "dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
 ```
 
+## Install MySQL 
+
+Alternate to Postgress you also can store your data in a MySQL/MariaDB database.
```
Comment (justinegeffen) [inline website/site/content/download/personal-edition/ubuntu.md:152]:
```suggestion
As an alternative to Postgres, you also can store your data in a MySQL/MariaDB database.
```

File: website/site/content/download/personal-edition/ubuntu.md [hunk 5]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
...
+To install, run:
+
+```
+sudo apt install mariadb-server mariadb-client
+```
+
+Log in as root in your database:
```
Comment (justinegeffen) [inline website/site/content/download/personal-edition/ubuntu.md:159]:
```suggestion
Log in as `root` in your database:
```

File: website/site/content/download/personal-edition/ubuntu.md [hunk 6]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
...
+```
+sudo mysql 
+```
+
+On the MySQL prompt, run the following commands (**change the user/password** to your own values):
```
Comment (justinegeffen) [inline website/site/content/download/personal-edition/ubuntu.md:164]:
```suggestion
At the MySQL prompt, run the following commands (change `user/password` to your own values):
```

File: website/site/content/download/personal-edition/ubuntu.md [hunk 7]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
...
+<pre>
```
Comment (justinegeffen) [inline website/site/content/download/personal-edition/ubuntu.md:165]: I'm not familiar with `<pre>` - is it typed like that in the command prompt/terminal?

File: website/site/content/download/personal-edition/ubuntu.md [hunk 8]
```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
...
+GRANT ALL on boards.* to <b>'boardsuser'@'localhost'</b> identified by '<b>boardsuser-password</b>';
+
+</pre>
+
+Exit the mysql-prompt:
+```
+exit
+```
+
+Edit the Focalboard config.json:
```
Comment (justinegeffen) [inline website/site/content/download/personal-edition/ubuntu.md:176]:
```suggestion
Edit the Focalboard `config.json`:
```

**Review by justinegeffen — COMMENTED** [review]
This is great, thank you! I did a brief review of both docs for grammar and style and have made some comments. LMK if you have any questions! :)

File: website/site/content/download/personal-edition/ubuntu.md (hunk 7 above)
Comment (ctlaltdieliet) [reply website/site/content/download/personal-edition/ubuntu.md:164]: no, it's bad HTML 
should be replaced with ```

**Review by ctlaltdieliet — COMMENTED** [review]
LGTM

**Review by justinegeffen — APPROVED** [review]

File: website/site/content/download/personal-edition/ubuntu.md (hunk 7 above)
Comment (jespino) [reply website/site/content/download/personal-edition/ubuntu.md:164]: this `<pre>` is still not replaced.

File: website/site/content/download/personal-edition/ubuntu.md [hunk 9]
```diff
@@ -147,6 +146,42 @@ Change the dbconfig setting to use the postgres database you created:
...
+```
+
+Edit the Focalboard `config.json`:
+
+```
+nano /opt/focalboard/config.json
+```
+
+Change the dbconfig setting to use the MySQL database you created:
+"dbtype": "mysql",
```
Comment (jespino) [inline website/site/content/download/personal-edition/ubuntu.md:181]: Here are missing the 3 backticks

File: website/site/content/download/personal-edition/ubuntu.md [hunk 10]
```diff
@@ -147,6 +146,42 @@ Change the dbconfig setting to use the postgres database you created:
...
+"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
+```,
```
Comment (jespino) [inline website/site/content/download/personal-edition/ubuntu.md:183]: And I think this `,` here is not necesary, also the code block is going to be there and the `,` is going to be in the next line, I would remove that.

**Review by jespino — CHANGES_REQUESTED** [review]
Another tiny detail required

**Review by jespino — APPROVED** [review]
Now looks good to me. Thanks!

**Pull request merged**